app/controllers/employments_controller.rb

```diff
@@ -0,0 +1,100 @@
+# JSON API controller that serves GSIs resource
+class EmploymentsController < ApplicationController
+  respond_to :json
+  before_filter :assign_model
+  after_action :verify_authorized, except: :index
+
+  include JsonControllerHelper
+
+  # retreives the parent model and the association from the DB
+  def assign_model
+    @course = policy_scope(Course).find(params[:course_id])
+    @model = EmploymentPolicy::Scope
+      .new(current_user, Employment).resolveEmployments(@course)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.